src/risedevtool/src/compose.rs

```diff
@@ -14,10 +14,11 @@
 
 //! Generate docker compose yaml files for risedev components.
 
-const DOCKER_COMPUTE_NODE_IMAGE: &str = "ghcr.io/singularity-data/compute-node:latest";
```

Comment:
why do we need an individual risedev tool instead of calling `make docker-compose` in `Makefile.toml`

Reply:
risedev generates the docker-compose file